core-icon.html

```diff
@@ -167,7 +167,11 @@
         }
 
         this.iconset = this.computeIconset(this.icon);
-        this.iconsetElement.applyIcon(this.root, this.iconName, this.theme);
+        if (this.iconsetElement) {
+          this.iconsetElement.applyIcon(this.root, this.iconName, this.theme);
+        } else {
+          console.warn('iconset "' + this.iconset + '" not found!');
```

Comment:
I agree though: defaulting to the `"icons"` iconset makes core-icon assume it can always load icons. If you want to bind to the `icon` of one of these though, it will get an empty string or null. This whole block should be skipped in that case.